### DISCUSSION TITLE
How to export the graphics?

### DISCUSSION THREAD

**Discussion by FrancoPaez** [plouc/nivo#1664] [Q&A]
Hi! I was reading the documentation but I was left with doubts.

Is there a way to download the ResponsivePie, ResponsiveLine and ResponsiveBar graph?

Or do I have to use its version in canvan

is there a hack in that case?

extra: can it only be downloaded with .img extension or is there a possibility to do it .pdf?

Thank you!

**Comment by wyze** [comment]
You don't have to use the canvas version to download. Here is how I do it at work with any chart type. https://codesandbox.io/s/dank-wood-tgrfw?file=/src/App.tsx

**Reply by fabio-carvalho88** [reply]
is it possible to group a couple of charts inside a downloadable pdf?

**Reply by Rhysjosmin** [reply]
thanks man this saved me
